packages/react-native/React/DevSupport/RCTPausedInDebuggerOverlayController.mm

```diff
@@ -10,28 +10,29 @@
 
 #import "RCTPausedInDebuggerOverlayController.h"
 
-@interface RCTPausedInDebuggerViewController : UIViewController
+@interface RCTPausedInDebuggerViewController : UIViewController // [macOS]
```

Comment:
Isn't this the same as upstream? Why do we have `// [macos]` here?

Reply:
At some point I had RCTUIViewController, should revert, or actually introduce that macro in.